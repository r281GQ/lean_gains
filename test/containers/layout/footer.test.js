import React from 'react';
import { shallow } from 'enzyme';

import Footer from './../../../src/containers/layout/footer';

describe('Header test', () => {
  it('should render', () => {
    const component = shallow(<Footer />);

    expect(component).toHaveLength(1);
    expect(component).toMatchSnapshot();
  });
});
