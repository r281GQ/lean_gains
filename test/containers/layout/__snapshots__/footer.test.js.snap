// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header test should render 1`] = `
<div
  className="footer__container"
>
  <Paper
    circle={false}
    className="footer__paper"
    rounded={true}
    style={
      Object {
        "backgroundColor": "#E0E0E0",
      }
    }
    transitionEnabled={true}
    zDepth={2}
  >
    <IconButton
      disableTouchRipple={false}
      disabled={false}
      href="https://endrevegh.wordpress.com/"
      iconStyle={Object {}}
      target="_blank"
      tooltip="WordPress"
      tooltipPosition="top-center"
      touch={false}
    >
      <FontIcon
        className="fa fa-wordpress"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      />
    </IconButton>
    <IconButton
      disableTouchRipple={false}
      disabled={false}
      href="https://github.com/r281GQ/lean_gains"
      iconStyle={Object {}}
      target="_blank"
      tooltip="GitHub"
      tooltipPosition="top-center"
      touch={false}
    >
      <FontIcon
        className="fa fa-github"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      />
    </IconButton>
  </Paper>
</div>
`;
